Update search feature v1.2

sylviaokt opened the pull request:
Fixes #6

sylviaokt committed:
Remove FindCommand

sylviaokt committed:
Update Search feature for v1.2

internityz commented on src/main/java/seedu/address/model/person/NameAndSchoolContainsKeywordsPredicate.java:
```diff
@@ -0,0 +1,32 @@
+    }
+
+    @Override
+    public boolean test(Person person) {
```
would this cause a problem where name will be searched when user searched for a student's school

internityz reviewed (commented):
LGTM just a question for the predicate

sylviaokt committed:
Update search feature for v1.2

Changed the implementation for the search feature
such that the keywords for name will only be matched
to name, and the keywords for school will only be matched
to school.

internityz reviewed (approved):
LGTM
